test/dotcom_web/components/system_status/subway_status_test.exs

```diff
@@ -9,13 +9,167 @@ defmodule DotcomWeb.Components.SystemStatus.SubwayStatusTest do
   alias Dotcom.SystemStatus.Subway
   alias Test.Support.Factories
 
+  @lines_without_branches List.delete(Subway.lines(), "Green")
+
   setup :verify_on_exit!
 
   setup do
     stub_with(Dotcom.Utils.DateTime.Mock, Dotcom.Utils.DateTime)
     :ok
   end
 
+  describe "homepage_subway_status/1" do
+    test "renders all rows as 'Normal Service' when there are no alerts" do
+      # Setup
+      alerts = []
+
+      # Exercise
+      rows = status_rows_for_alerts(alerts)
+
+      # Verify
+      assert rows
+             |> Enum.each(fn row ->
+               assert status_label_text_for_row(row) == "Normal Service"
+             end)
+    end
+
+    test "shows all of the route pills by default" do
+      # Setup
+      alerts = []
+
+      # Exercise
+      rows = status_rows_for_alerts(alerts)
+
+      # Verify
+      assert rows
+             |> Enum.each(fn row ->
```

Comment:
`Enum.each` always returns `:ok`, maybe this should be `Enum.all?`?

Reply:
Ahh oops - there's an `assert` _inside_ the loop as well. The outer layer doesn't need an `assert`, I think.

[PTALA](https://github.com/mbta/dotcom/pull/2404/commits/7275be233a83d0c9529aa329da843c4ba05184d7)